### PR TITLE
Fix up for Issue #418

### LINKED ISSUE #418: 3.8.2 Media flow management - delete suggestion for flipping SDP direction?
- Section 3.8.2 - "the media flow from these sessions can be managed by
specifying SDP direction attributes in the descriptions". This seems out
of place. Surely flipping track.muted is a much simpler way of managing
them if the problem is what to play to the user?

Suggestion: Remove text.

### PR DESCRIPTION
**Pull request by fluffy**
Fixes #418

### CODE REVIEW

**Comment by juberti** [comment]
Since the goal is to only have one session active at a time, I think the best way of handling this would be to use transceiver.setDirection (rather than muted, which requires all sessions to be active, and some receive silent media)

**Comment by juberti** [comment]
Right, but I think that matches the desired outcome best.

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -945,8 +945,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           will end up with N parallel PeerConnection sessions, each
           with a local and remote description and their own local and
           remote addresses. The media flow from these sessions can be
-          managed by specifying SDP direction attributes in the
-          descriptions, or the application can choose to play out the
+          managed, or the application can choose to play out the
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: What does managed mean here?

**Comment by juberti** [comment]
Is this now OBE by #472?

**Pull request closed without merging**